recipes/sagemaker-code-editor/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+# This code includes code from https://github.com/conda-forge/openvscode-server-feedstock
+# which is licensed under the BSD-3-Clause License.
+
+export NODE_OPTIONS=--openssl-legacy-provider
+
+# ripgrep tries to download binary from github, copying the ripgrep executable to the cache location so that it doesn't do that
+mkdir /tmp/vscode-ripgrep-cache-1.15.5
+mkdir /tmp/vscode-ripgrep-cache-1.15.6
+cp "${PREFIX}/bin/rg" /tmp/vscode-ripgrep-cache-1.15.5/
+cp "${PREFIX}/bin/rg" /tmp/vscode-ripgrep-cache-1.15.6/
```

Comment:
Are these specific versions expected to match with the ones added in `meta.yaml`?

Reply:
To add some background here - if we just rely on the node vscode-ripgrep dependency, it tries to download a binary from github and usually fails. So, we're copying the conda forge `rg` binary to the cache location so that it doesn't do that. These versions (1.15.5, 1.15.6 ) of vscode-ripgrep are used by different versions of Code - OSS such as [here](https://github.com/microsoft/vscode/blob/1.83.1/package.json#L74)

I do see issues with current implementation though - the conda forge ripgrep version could be something other than required (and could possibly break functionality) - thinking more on this ....

Reply:
I'll update this implementation to not rely on hardcoded versions of ripgrep. We still need to replace the node ripgrep with the one from conda-forge - where there could be a version mismatch. That shouldn't be a blocker since we've verified that the needed functionality works. We'd like to proceed with this and identify a solution for it incrementally.